sessionize/lib/src/commonMain/kotlin/co/touchlab/sessionize/architecture/BasePub.kt

```diff
@@ -13,7 +13,7 @@ abstract class BasePub<T> : Pub<T> {
 
     fun applyError(throwable: Throwable) {
         val subIter = subs().iterator()
-        subs().clear()
+//        subs().clear()
```

Comment:
@kpgalligan found this on a Slack call while screensharing. We commented it out because it was preventing useful error messages. He can speak better to what it's doing specifically, but I know the fix worked.